opencti-platform/opencti-front/src/private/components/analysis/external_references/ExternalReferenceCreation.tsx

```diff
@@ -309,17 +307,7 @@ ExternalReferenceCreationProps
                       label={t('Associated file')}
                       FormControlProps={{ style: { marginTop: 20, width: '100%' } }}
                       InputLabelProps={{ fullWidth: true, variant: 'standard' }}
-                      InputProps={{
-                        classes: { fullWidth: true, variant: 'standard' },
-                        onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
-                          const fileName = event.target.value.split('\\').pop();
-                          const externalIdValue = (document.getElementById('external_id') as HTMLInputElement).value;
-                          if (!externalIdValue && fileName) {
-                            setFieldValue('file', event.currentTarget.files?.[0]);
-                            setFieldValue('external_id', fileName);
-                          }
-                        },
-                      }}
```

Comment:
Could you explain why all this code was removed ? What was it used for ?

Reply:
It was used to set an external_id with the name of the file if the external_id is not provided by the user. The problem is : 
The code is working fine if you don't put an external_id but not working if you put one.